src/eko/evolution_operator/__init__.py

```diff
@@ -21,7 +21,6 @@
 from ..kernels import non_singlet as ns
 from ..kernels import singlet as s
 from ..member import OpMember
-from ..scale_variations import expanded, exponentiated
```

Comment:
@felixhekhorn this fix doesn't work, since ``scale_variations.__init__`` doesn't contain ```import expanded, exponentiated```.  Would you prefer to revert or add them to the `__init__`?

Reply:
added imports in [88f1d6f](https://github.com/N3PDF/eko/pull/91/commits/88f1d6fd6ba5875a6d6085356eee98953d4aa5e9) - good you realized, still I got the green tick ...